md/release-notes.md

```diff
@@ -18,7 +18,8 @@
 ### 6.x forms based on GWT technology
 Studio forms based on Google Web Toolkit (GWT) technology are not supported anymore, starting with Bonita 7.8. They have been removed from Bonita Studio. Importing a .bos will not import such forms.
 Cloning a Git repository or migrating a SVN repository will remove such forms and their related content (validators, look'n'feels, forms and widgets templated).  
-Before you start Bonita Studio 7.8 or migrate your production to Bonita 7.8., make sure you have replaced such forms/pages by forms/pages created with more recent technologies and newer concepts, offered since Bonita 7.0: [UI Designer](ui-designer-overview.md) and [contract and context](contracts-and-contexts.md).
+Before you migrate your production to Bonita 7.8, make sure you use Bonita Studio to replace such forms/pages by forms/pages created with more recent technologies and newer concepts, offered since Bonita 7.0: [UI Designer](ui-designer-overview.md) and [contract and context](contracts-and-contexts.md).  
+Moreover, deploying a process (.bar file) containing forms/pages developped using Google Web Toolkit is not possible anymore.
```

Comment:
@NathalieC is this last sentence ok for you, or does it introduce confusion, because we never acknowledged in the past that on .bar file generated with an older Studio could be deployed on a more recent bundle?

Reply:
It's fine by me.

Reply:
Hi, my 2 cents: 
- Resources was a features used a lot by customers in v6 forms. You should list it in the brackets, it's going to have a big impact on our historical customers. Add "Resources, confirmation" and all the tabs names of the studio v6 forms that are going to disappear.  
- Our customers almost never do SVN backup when they migrate. Once the repo is migrated, there isn't any rollback possible. If everything of v6 forms is lost, a red warning should be added in this RN. Unless I don't understand correctly, I think this information is crucial and should not be drowned in the middle of a paragraph in the release note. I urge you to add a control in the Studio to prevent repo to be migrated if there is v6 forms. Our users will not expect data to be lost. Migrated and not used is one thing. Migrating a repo and having no rollback option if you forgot to migrate your forms, and loosing them, this is dangerous. 
- Maybe add this warning in workspaces-and-repositories too?

Reply:
@mariellebonita : By Customer Success and Support programs, we, at Product, have been told that all customers still using v6 forms have received a notice, already in 7.7, about the deprecation and then removal of v6 forms in Bonita starting in 7.8.
This is why our intention is to write this in the release note as a reminder to our already warned 26 customers still using v6 forms, and not bother all others too much.
On the other hand, I understand your concern about SVN management. I'll check with the Studio team how difficult it is now to merely prevent migration in case of SVN.

Reply:
OK to add a warning in the page you're pointing to. I'll do it.

Reply:
Got an answer from the Studio team:
Migrating a SVN repository automatically creates a tag (a saved copy/restauration point).
So nothing should be lost.
We can keep the current behavior: migrate what can be, and clearly state what has not been (v6 forms and case overview pages) and why (not supported anymore, processes and forms should be redesigned).

Reply:
@rbioteau: can you add all Studio features removed with the v6 forms removal, like "Resources" and the confirmation page after a form is submitted? This used to allow to search for the next taks within a case, which is not possible anymore with the current task list. This use case must be implemented with a Living Application.

Reply:
Thx. Now back to the original author, Manu, to complete the PR with those items.

Reply:
Made an addition to Manu's PR with the content provided by Romain + link to "migrate a form from 6.x" to address the configuration page removal.